src/pages/post_download.tsx

```diff
@@ -89,9 +91,18 @@ export function getRunningInstructions(platform) {
     if (platform === 'win') {
        return <p>First, make sure <a href="https://adoptium.net/download/">Java</a> is installed on your computer. Then, unzip the archive, and double-click on <code>openrefine.exe</code> or <code>refine.bat</code> if the former does not work.</p>
     } else if (platform === 'win-with-java') {
-       return <p>Unzip the archive, and double-click on openrefine.exe or refine.bat if the former does not work.</p>
+       return <p>Unzip the archive, and double-click on <code> openrefine.exe</code> or <code>refine.bat</code> if the former does not work.</p>
     } else if (platform === 'mac') {
-       return <p>Open the downloaded DMG file, drag icon into the <code>Applications</code> folder and double click on it. <Link to="docs/manual/installing#install-or-upgrade-openrefine">See our documentation for how to circumvent security warnings</Link> when opening the DMG file.</p>;
+      return (
+        <p>Open the downloaded DMG file, drag the icon into the <code>Applications</code> folder, and double-click on it. 
+          <p className='alert margin-top--md'> 
+          <strong style={{color:'red'}}> Important:</strong> See our documentation for {' '}
```

Comment:
We typically would not use red color to mark importance, even for security.  We typically reserve red color only for true danger (potential loss of data, etc.)  In this case, we are guiding the user on how to circumvent built-in security of OS's.  So I'd probably make this the same color as the INFO color in Docusaurus `--ifm-color-info-dark` currently `#4cb3d4` or `--ifm-color-info-light`.

FYI, overrides and themes are in https://github.com/OpenRefine/openrefine.org/blob/master/src/css/custom.css
But in this case of our custom `.tsx` file, the `style=` is fine, just change the color.

Also, you can see examples of some [admonitions](https://docusaurus.io/docs/markdown-features/admonitions) we use under the [Running](https://openrefine.org/docs/manual/running) section of our documentation.

Reply:
The deploy preview looks good to me now.
https://deploy-preview-418--openrefine-website.netlify.app/post_download?version=3.8.7&platform=mac